specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -7802,6 +7778,33 @@
       },
       "description": "The language codes supported for input by OcrSkill."
     },
+    "OcrSkillLineEnding": {
+      "type": "string",
+      "enum": [
+        "Space",
+        "CarriageReturn",
+        "LineFeed"
```

Comment:
Can these two be used together (i.e. line endings on Windows/DOS)? If so, how are values combined (delimiter)?

Reply:
See next comment below...

Reply:
(But the answer is no - currently this is an enum where you choose one value.)